.github/workflows/ci.yml

```diff
@@ -82,6 +82,39 @@ jobs:
       env:
         DOCKER_DEFAULT_PLATFORM: ${{ matrix.platform }}
 
+  # Run pathogen repo CI builds with the final image
+  test-pathogen-repo-ci:
+    needs: build
+    strategy:
+      # XXX TODO: Test on multiple platforms via the matrix too, as above?
+      matrix:
+        include:
+          - { pathogen: avian-flu,       build-args: auspice/flu_avian_h5n1_ha.json }
+          - { pathogen: ebola }
+          - { pathogen: lassa }
+          - { pathogen: monkeypox }
+          - { pathogen: mumps }
+          - { pathogen: ncov,            build-args: all_regions -j 2 --profile nextstrain_profiles/nextstrain-ci }
+          - { pathogen: seasonal-flu,    build-args: --configfile profiles/ci/builds.yaml -p }
+          - { pathogen: zika }
+    name: test-pathogen-repo-ci (${{ matrix.pathogen }})
+    uses: nextstrain/.github/.github/workflows/pathogen-repo-ci.yaml@master
+    with:
+      repo: nextstrain/${{ matrix.pathogen }}
+      build-args: ${{ matrix.build-args }}
+      env: |
+        NEXTSTRAIN_DOCKER_IMAGE: ghcr.io/nextstrain/base:${{ needs.build.outputs.tag }}
+
+    # XXX FIXME: Jobs which call reusable workflows (uses: …) can't specify
+    # continue-on-error.¹  That's unfortunate, as we don't want to block
+    # subsequent jobs or fail the workflow because of these.  They're more
+    # smoke tests than gating tests.  Would be good to figure out how to adjust
+    # subsequent job conditions (if: …) to get ~equivalent behaviour.
```

Comment:
If the pathogen-repo-ci tests shouldn't block the remaining jobs, can it just be forced to be the last step of the workflow?

Reply:
Ah, the comment is maybe a bit misleading because I forgot to update it to remove the "block subsequent jobs" bit. No subsequent jobs actually depend on (`needs: …`) the `test-pathogen-repo-ci` jobs, so failures of the latter won't block any jobs. But failures will still fail the workflow as a whole, which seems possibly undesirable.

Reply:
[Reworded](https://github.com/nextstrain/docker-base/compare/f351e4c8aab40acdd1878ba5bd51586d49b9409b..12000a20d9df02b59cce4aa7f466833fb54df362)

Reply:
Ah, I see. I think the workflow failing as a whole is a good "notification" of these pathogen-repo-ci failures or else we might miss them. 

Although, thinking aloud on maybe a way around the whole workflow failure:
Add `continue-on-error: ${{ inputs.continue-on-error }}` to the reusable pathogen-repo-ci workflow's `build` job. Then workflows that use the pathogen-repo-ci workflow can pass the input to force the `continue-on-error` behavior.

Reply:
Ahh, yes, that might work!

Reply:
I agree ok to leave them as failing the workflow for now. If we find they're too noisy/uninformative after some time, we can revisit making them smoke-tests only.

Reply:
> Ah, I see. I think the workflow failing as a whole is a good "notification" of these pathogen-repo-ci failures or else we might miss them.

[I implemented](https://github.com/nextstrain/.github/pull/40) the `continue-on-error` pass thru, and the UI that GitHub presents makes it [pretty hard to miss the failures](https://github.com/nextstrain/.github/pull/40#issuecomment-1540545735). So that's nice!